src/textmate/src/bicep.ts

```diff
@@ -58,6 +58,7 @@ const keywords = [
   'existing',
   'import',
   'as',
+  'type',
```

Comment:
Please make the same change to the [highlightjs](https://github.com/Azure/bicep/blob/main/src/highlightjs/src/bicep.ts) & [monarch](https://github.com/Azure/bicep/blob/main/src/monarch/src/bicep.ts) grammars!